src/editor/Editor.js

```diff
@@ -89,6 +89,7 @@ define(function (require, exports, module) {
         LINEWISE_COPY_CUT   = "lineWiseCopyCut",
         SCROLL_PAST_END     = "scrollPastEnd",
         SHOW_CURSOR_SELECT  = "showCursorWhenSelecting",
+        SHOW_FULL_FILEPATH  = "showFullFilePathInTopBar",
```

Comment:
Can you add this preference in DocumentCommandHandler.js only, since this preference is not used in Editor.js.
Also changing the preference name to showFullFilePath would be more generic compared to showFullFilePathInTopBar.